hvac/api/secrets_engines/identity/mfa/okta.py

```diff
@@ -0,0 +1,137 @@
+from hvac import utils
+from hvac.api.secrets_engines.identity.mfa.mfa_method_mixin import MfaMethodMixin
+from hvac.constants.identity import DEFAULT_MOUNT_POINT
+
+
+class Okta(MfaMethodMixin):
+    method_type = "okta"
+
+    def create(
+        self,
+        org_name,
+        api_token,
+        method_name=None,
+        username_format=None,
+        base_url=None,
+        primary_email=None,
+        use_passcode=None,
+        mount_point=DEFAULT_MOUNT_POINT,
+    ):
+        """
+        Create an MFA Method of method_type Okta.
+
+        Supported methods:
+            POST: /{mount_point}/mfa/method/okta. Produces: 204 (empty body)
+
+        :param org_name: Name of the organization to use in the Okta API.
+        :type org_name: str | unicode
+        :param api_token: Okta API key.
+        :type api_token: str | unicode
+        :param method_name: Unique name identifier for this MFA method.
+            Supported from Vault 1.13.0.
+        :type method_name: str | unicode
+        :param username_format: Format string for mapping identity names to
+            MFA method names. Values to substitute should be placed in `{{}}`.
+            For example, `{{identity.entity.name}}@example.com`.
+            If blank, the Entity's Name field is used as-is.
+        :type username_format: str | unicode
+        :param base_url: If set, will be used as the base domain for API requests.
+            Examples: okta.com, oktaprevew.com, okta-emea.com
+        :type base_url: str | unicode
+        :param primary_email: If set, the username will match the Okta profile's
+            primary_email instead of the Okta profile's login.
+        :type primary_email: bool
+        :param use_passcode: If set, the user is reminded to use the passcode
+            upon MFA validation.
+        :type use_passcode: bool
```

Comment:
This will be introduced in a future version of vault once this PR is merged in vault:
https://github.com/hashicorp/vault/pull/24563

Reply:
This will land in vault 2.17